.github/workflows/pr_validation.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
     runs-on: ${{matrix.os}}
 
     strategy:
+      fail-fast: false
```

Comment:
This should stop GitHub from signalling a job cancel to subsequent matrix jobs if a job failed